admin/client/src/components/Breadcrumb/Breadcrumb.js

```diff
@@ -38,4 +39,10 @@ Breadcrumb.propTypes = {
   crumbs: React.PropTypes.array,
 };
 
-export default Breadcrumb;
+function mapStateToProps(state) {
```

Comment:
ReactJS components placed in `components/Breadcrumb` should not be tied to Redux state, this should be fed into the component by the "parent" container component.